src/Okta.Sdk.UnitTests/UsersClientShould.cs

```diff
@@ -124,5 +125,464 @@ public async Task RemoveApplicationTargetFromApplicationAdministratorRoleForUser
+                                          ""action"": ""NONE""
+                                        }
+                                      }
+                                    },
+                                    ""accountLink"": {
+                                      ""filter"": null,
+                                      ""action"": ""AUTO""
+                                    },
+                                    ""subject"": {
+                                      ""userNameTemplate"": {
+                                        ""template"": ""idpuser.userPrincipalName""
+                                      },
+                                      ""filter"": null,
+                                      ""matchType"": ""USERNAME""
+                                    },
+                                    ""maxClockSkew"": 0
+                                  },
+                                  ""_links"": {
+                                    ""authorize"": {
+                                      ""href"": ""https://${yourOktaDomain}/oauth2/v1/authorize?idp=0oa62b57p7c8PaGpU0h7&
+                                          client_id={clientId}&response_type={responseType}&response_mode={responseMode}&
+                                          scope={scopes}&redirect_uri={redirectUri}&state={state}"",
+                                      ""templated"": true,
+                                      ""hints"": {
+                                        ""allow"": [
+                                          ""GET""
+                                        ]
+                                      }
+                                    },
+                                    ""clientRedirectUri"": {
+                                      ""href"": ""https://${yourOktaDomain}/oauth2/v1/authorize/callback"",
+                                      ""hints"": {
+                                        ""allow"": [
+                                          ""POST""
+                                        ]
+                                      }
+                                    },
+                                    ""idpUser"": {
+                                        ""href"": ""https://${yourOktaDomain}/idps/0oa62b57p7c8PaGpU0h7/users/00ub0oNGTSWTBKOLGLNR"",
+                                        ""hints"": {
+                                          ""allow"": [
+                                            ""GET"",
+                                            ""DELETE""
+                                          ]
+                                        }
+                                      }
+                                  }
+                                }]";
+            var mockRequestExecutor = new MockedStringRequestExecutor(rawResponse);
+            var client = new TestableOktaClient(mockRequestExecutor);
+            await client.Users.ListUserIdentityProviders("foo").ToListAsync();
```

Comment:
I suggest also verifying that it returns a list with the object we are expecting.